src/common/tools.ts

```diff
@@ -106,3 +106,11 @@ export const getLabelFromValueInOptionList = <T>(
   const target = optionList.find(({ value }) => value === targetValue)
   return target?.label || ''
 }
+
+export const commonTimeUnits = [
+  { value: 'ms', label: 'ms' },
+  { value: 's', label: 's' },
+  { value: 'm', label: 'm' },
+  { value: 'h', label: 'h' },
+  { value: 'd', label: 'd' },
+]
```

Comment:
Why not this?

```
export const commonTimeUnits = ['ms', 's', 'm', 'h', 'd']
```

Reply:
I think it should be a problem of packaging in the component?

But I should provide options for both

`Label` and `Value` or `plain string arrays`, mainly to help in writing such data structures